### PR TITLE
[CON-405] Syncing up skeleton branch with main

### PR DESCRIPTION
**Pull request by plauche**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,30 @@
+## System Requirements
+
+Arcas load testing depends on GStreamer version 1.18.X being installed on your computer. If you do not have GStreamer installed you can use the following links to download:
+
+**NOTE**: Only GStreamer versions 1.18.X are compatible with this demo at this time.
+
+- Linux - Please see the GStreamer installation [guide](https://gstreamer.freedesktop.org/documentation/installing/on-linux.html?gi-language=c)
+
+- MacOS will need to install both of the following:
```
Comment (ddimaria) [inline README.md:9]: What about M1?

Comment (plauche) [reply README.md:9]: @ddimaria that is a good question. I was syncing up this copy of the README with the copy in _main_ and used in the blog post, which doesn't give special instructions for M1 machines. I can ask Paul to test the instructions on his mac and see if we need to do anything special. However I think that should addressed in a separate follow up issue as the scope is a bit bigger than just syncing up this branch.

Comment (ddimaria) [reply README.md:9]: I agree, but figured I'd make a note somewhere.  They don't seem to have prebuilt packages for M1.

File: README.md [hunk 2]
```diff
@@ -86,31 +108,31 @@ await deviceLoaded(SOCKET_URI, async (device) => {
       await session.stop();
       process.exit();
     }
-  }, TEST_INTERVAL_S);
+  }, TEST_INTERVAL_S * 1000);
```
Comment (kyleparrott) [inline README.md:111]: 🎉

**Review by kyleparrott — APPROVED** [review]

**Pull request merged**